drivers/boston_project_scorecard/app/models/boston_project_scorecard/report.rb

```diff
@@ -289,15 +289,15 @@ def send_email_to_owner
         'Question 22',
         'Question 23',
       ]
-      generator = HudApr::Generators::Apr::Fy2023::Generator
+      generator = HudApr.current_generator(report: :apr)
```

Comment:
Hmm, this might be ok, but it means if the APR changes the underlying report uses a different generator which may change cell numbers, which would mean the report would run, but could provide unintended results.  Not sure if I like that or not.  I guess it's one fewer thing to update, but also one more possibility of confusion.